src/parser/rogue/assassination/modules/core/ComboPoints.tsx

```diff
@@ -12,66 +12,66 @@ class ComboPoints extends Analyzer {
     comboPointTracker: ComboPointTracker,
   };
 
-  makeExtraSuggestion(spell) {
-    return <>Avoid wasting combo points when casting <SpellLink id={spell.id} />. Combo points for Seal Fate are not considered. </>;
+  protected comboPointTracker!: ComboPointTracker;
+
+  makeExtraSuggestion(spellId: number) {
+    return (
+      <>
+        Avoid wasting combo points when casting <SpellLink id={spellId} />. Combo points for Seal
+        Fate are not considered.{' '}
+      </>
+    );
   }
 
-  suggestions(when) {
+  suggestions(when: any) {
```

Comment:
I believe Dambroda merged a change to type out suggestions, so `suggestions(when: When)`

and import { When } from 'parser/core/ParseResults';